stories/voice-search.stories.js

```diff
@@ -146,13 +146,12 @@ storiesOf('VoiceSearch', module)
           left: 0;
           right: 0;
           z-index: 2;
-          display: flex;
```

Comment:
Squash this in the previous commit, to simplify the history

Reply:
We squash all PRs as a single commit on merging to avoid many rebases on working branches. This allows for making the commits within the branch however necessary without needing to think about history 🙂

Reply:
@BenoitPerrot This whole PR is going to be squashed on merge.

Reply:
OK clear! never mind then!

Reply:
(Why not squashing _before_ pushing the review then?)

Reply:
This is more work for not really much benefit we decided, especially for third-party contributors. Especially since we can enforce a squash commit in the settings of the repository 🙂